<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Snake 2</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background:rgb(138, 226, 52)
}</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>550</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
	background:white;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background:white;
}</string>
    </property>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	background:rgb(66, 134, 244)
}</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetSize" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QSlider" name="horizontalSliderArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxX">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>49</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	background:white;
}</string>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelCoordinates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Set value for grid size:</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderDifficulty">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelDifficultyState">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Easy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelDifficulty">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Difficulty:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seed (number):</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxWalls">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Solid walls</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonPause">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	background:rgb(138, 226, 52)
}</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetTime" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QWidget{
	background:white;
}</string>
    </property>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Time passed:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberTimeMinutes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber {
	background:rgb(0,139,139)
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinutes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Minutes</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSeconds">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Seconds</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberTimeSeconds">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber {
	background:rgb(66, 134, 244)
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetScore" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background:white;
}</string>
    </property>
    <widget class="QLabel" name="labelScore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHighscore">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Highscore:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberScore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber {
	background:rgb(66, 134, 244)
}</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberHighscore">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber {
	background:rgb(255, 0, 0)
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScoreInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Scores:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelGameStatus">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGameOver">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>550</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:white;
	border:none;
	background:red
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderArea</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderArea</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>891</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
